recipes/strling/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "strling" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "6d3aff31de690b47f19dec31617185e7d3547b05d46ccb5a4cf01b3fb5b8c0e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/quinlan-lab/STRling/archive/614a48877a32f5b2fd628df7a56ce0dfe54bc8df.zip
+    sha256: {{ sha256 }}
+  - url: https://nim-lang.org/download/nim-1.0.6.tar.xz
+    sha256: 134e5751cf62fa189f76616a698ce0df4b4a0c1419ec0b582924087e585f66b3
+    folder: nimsrc
+
+build:
+  number: 0
+  skip: True  # [osx]
```

Comment:
You’re still skipping the macOS build. It is a bit confusing because the test on macOS is marked as passing, but that is only because it successfully detects that it has nothing to do.

Reply:
since nim is not available for macos and via conda and it cannot be compiled, we keep skipping